llvm/test/MC/AsmParser/sse2avx-intel.s

```diff
@@ -0,0 +1,88 @@
+# RUN: llvm-mc -triple x86_64 -x86-asm-syntax=intel -x86-sse2avx %s | FileCheck %s
```

Comment:
Perhaps merge sse2avx-att.s sse2avx-intel.s. Keep the att version.

Add `llvm-mc -triple=x86_64 -output-asm-variant=1 %s | llvm-mc -triple=x86_64 -x86-asm-syntax=intel -x86-sse2avx`
to test Intel syntax input.

Reply:
Done